simplq/src/main/java/me/simplq/controller/QueueController.java

```diff
@@ -69,7 +78,7 @@
     } else if (queueName != null) {
       return ResponseEntity.ok(queueService.getQueueStatusByName(queueName));
     } else {
-      return new ResponseEntity(HttpStatus.BAD_REQUEST); // Todo Give reason
+      return ResponseEntity.badRequest().build();
```

Comment:
Can you add back that todo comment? Or we should give a reason for the client to to understand that he has to send in either the queueName or queueId as query param.